lms/static/js/learner_dashboard/views/course_entitlement_view.js

```diff
@@ -379,7 +378,7 @@ class CourseEntitlementView extends Backbone.View {
     const dateFormat = moment.localeData().longDateFormat('L').indexOf('DD') >
       moment.localeData().longDateFormat('L').indexOf('MM') ? 'MMMM D, YYYY' : 'D MMMM, YYYY';
 
-    return _.map(formattedSessionData, (session) => {
+    return formattedSessionData.map((session) => {
```

Comment:
could probably also be cleaned up with spread or object assign

Reply:
Done.